exercises/allergies/uAllergyTests.pas

```diff
@@ -13,39 +13,39 @@   AllergyTests = class(TObject)
     procedure No_allergies_means_not_allergic;
 
```

Comment:
I can do that.  I hadn't bothered as I have not seen that in other tracks, namely C#.